plugins/env/app/controllers/environment_variable_groups_controller.rb

```diff
@@ -42,13 +42,14 @@ def destroy
   end
 
   def preview
+    @deploy_groups = !params[:scope].empty? ? DeployGroup.where(permalink: params[:scope]) : DeployGroup.all
```

Comment:
prefer a step-by-step approach

```Ruby
@deploy_groups = DeployGroup.all
@deploy_groups = @deploy_groups.where(permalink: params[:scope]) if params[:scope].present?
```

prefer calling it deploy_group so it's easier to add environment or other things for filtering